client/src/index.html

```diff
@@ -9,9 +9,21 @@
   <link rel="preconnect" href="https://fonts.gstatic.com">
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
 </head>
 <body>
   <app-root>
   </app-root>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/js/bootstrap.bundle.min.js" 
```

Comment:
I am not sure about all these line of code, I don't know what it does. Will read later about this.

Reply:
Yes, what is this for? Is this required (if it is, do you have a link to any website that explains this)?

I would normally assume that if you do `npm install ng-bootstrap` on `client` that should be enough, so it would be helpful if you could explain what this is for, thanks!